packages/types/src/userSettings.ts

```diff
@@ -104,6 +108,7 @@ export interface UserSettingsJSON extends ClerkResourceJSON {
   social: OAuthProviders;
 
   saml: SamlSettings;
```

Comment:
🔧 We should mark `saml` as deprecated in favor of `enterprise_sso`:
```suggestion
  /**
  * @deprecated Use `enterprise_sso` instead
  */
  saml: SamlSettings;
```

Reply:
Fixed [f9c2194](https://github.com/clerk/javascript/pull/4596/commits/f9c2194678113a36fdcbb84bb55042d170b9e230)